card/mod/search/set/abstract/wql_search.rb

```diff
@@ -8,7 +8,12 @@ def search args={}
   query.run
 end
 
+def cache_query?
+  true
+end
```

Comment:
I assume there is a new wikirate use case where we don't want to cache the query?

Reply:
It's an old wikirate use case that you broke with your search refactoring a while ago (when you introduced `wql_from_content` and started to cache the return value of `wql_hash`) but it didn't break any tests until I had to regenerate the seed data while I was struggling with a mess of failing tests.